new_jersey/ver2/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <title>New Jersey</title>
+</head>
+<body>
+<section class="container">
+    <section class="widget">
+        <div class="header">
+            <div class="refresh">
+                <img src="img/refresh.png" alt="refresh pic">
+            </div>
+        </div>
+        <div class="main">
+            <div class="main-pic">
+                <img src="img/sun'n_cloud.png" alt="sun and cloud pic">
+            </div>
+        </div>
+        <div class="forecast">
+            <div class="info">
+                <span class="temperature">79</span>
+                <img src="img/degree.png" alt="degree pic">
+            </div>
+            <div class="description_location">
```

Comment:
In BEM naming methodology, single underscore "_" is used to show that the style is a modifier, for example:
`<div class="button button_pressed"></div>`

If we need to use several words as a class name, it is advised to use "-": "description-location".